x-pack/plugins/security_solution/public/common/containers/sourcerer/constants.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const SOURCERER_FEATURE_FLAG_ON = false;
+
+export enum SOURCE_GROUPS {
+  default = 'default',
+  host = 'host',
+  detections = 'detections',
+  timeline = 'timeline',
+  network = 'network',
+}
+
+export type SourceGroupsType = keyof typeof SOURCE_GROUPS;
+
+export const sourceGroups = {
+  [SOURCE_GROUPS.default]: [
+    'apm-*-transaction*',
+    'auditbeat-*',
+    'endgame-*',
+    'filebeat-*',
+    'logs-*',
+    // 'packetbeat-*',
```

Comment:
Would like to know if there's a specific reason we comment out packetbeat here?

Reply:
I wanted to create a state where one of the defined indexes was not here. I can remove it if confusing. Since this is all commented out with the feature flag, we will have time to fix it. Really just left it so I'd have it later

Reply:
No problem, let's keep it like this.